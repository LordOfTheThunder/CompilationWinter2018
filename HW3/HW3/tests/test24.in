void tomato() {
	int x;
	x = potato();
}
