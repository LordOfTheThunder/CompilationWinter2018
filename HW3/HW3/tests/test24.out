line 3: function potato is not defined
